pybossa/forms/forms.py

```diff
@@ -267,6 +267,9 @@ def in_range(self):
         return self.min_seconds <= minutes*60 + seconds <= self.max_minutes*60
 
 
+class ProgressReminderForm(Form):
```

Comment:
If we decide to rename the page, we may want to consider renaming the class (and all instances of the keyword `ProgressReminder`) to `TaskNotification`, etc.

Reply:
Yes, will do rephrase here and other places.